File: Fundamentos de Programación/Ejercicio 3.12.py
c = 0
sp = 0
si = 0
while c <= 200:
    if c%2 == 0:
        sp += c
    else:
        si += c
    c += 1
print('El resultado de la suma par es de',sp)
print('El resultado de la suma impar es de',si)

File: Fundamentos de Programación/Ejercicio 3.12.cpp
#include <iostream>

int main(){
    std::cout<<"3.12\n";
    int sp = 0, si = 0, c = 0;
    while(c<=200){
        if(c%2 == 0){
            sp += c;
        }
        else{
            si +=c;
        }
        c++;
    }
    std::cout<<"El resultado de la suma par es de "<<sp<<"\n";
    std::cout<<"El resultado de la suma impar es de "<<si<<"\n";
}